app/Http/Resources/AlbumCollection.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Http\Resources;
+
+use Illuminate\Http\Resources\Json\ResourceCollection;
+
+class AlbumCollection extends ResourceCollection
+{
+    public static $wrap = 'albums';
+
+    /**
+     * Transform the resource collection into an array.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return array|\Illuminate\Contracts\Support\Arrayable|\JsonSerializable
+     */
+    public function toArray($request)
+    {
+        return parent::toArray($request);
+    }
```

Comment:
https://laravel.com/docs/9.x/eloquent-resources#customizing-the-underlying-resource-class